examples/kubernetes.json

```diff
@@ -13,7 +13,8 @@
       {
         "name": "agentpool1",
         "count": 2,
-        "vmSize": "Standard_D2_v2"
+        "vmSize": "Standard_D2_v2",
+        "acceleratedNetworkEnabled": true

```

Comment:
This example model is meant to be a "vanilla" Kubernetes cluster with no optional features turned on. I think we shouldn't specify `acceleratedNetworkEnabled` here and instead add a new example file with acceleratedNetworkEnabled enabled

Reply:
How can I include this new example in the tests ?

Reply:
I'll take care of adding it to our Jenkins test suite. We don't test each individual feature on PRs since acs-engine has so many customization options, it would take forever to merge a PR if we had a test for all the different combinations of customizations.